src/core/Microsoft.PowerToys.Settings.UI.UnitTests/ModelsTests/BasePTModuleSettingsTest.cs

```diff
@@ -21,6 +24,21 @@ public class BasePTModuleSettingsTest
         [Obsolete]
         public void ToJsonString_ShouldReturnValidJSONOfModel_WhenSuccessful()
         {
+            //Mock Disk access
+            string saveContent = string.Empty;
+            string savePath = string.Empty;
+            var mockIOProvider = new Mock<IIOProvider>();
+            mockIOProvider.Setup(x => x.WriteAllText(It.IsAny<string>(), It.IsAny<string>()))
+                          .Callback<string, string>((path, content) =>
+                          {
+                              savePath = path;
+                              saveContent = content;
+                          });
+            mockIOProvider.Setup(x => x.ReadAllText(It.Is<string>(x => x.Equals(savePath, StringComparison.Ordinal))))
+                          .Returns(() => saveContent);
+
+            var settingsUtils = new SettingsUtils(mockIOProvider.Object);
+
```

Comment:
Can `GetMockIOProviderForSaveLoadExists` be used here?

Reply:
Yep good call 👍